variables.tf

```diff
@@ -80,3 +79,8 @@ variable "name_prefix" {
   type        = string
   description = "A prefix to add to the names of all created resources."
 }
+
+variable "interface_endpoint_ingress_sg" {
```

Comment:
We'll need to provide an illustrative example and run `make terraform/docs`.

Reply:
Dan, for the purpose of showing the interface endpoint being configured, will you prefer I modify the complete examples in the https://github.com/Datatamer/terraform-aws-networking/tree/master/examples/complete or create a new example showing how to create a security group and pass the security group ID to configure the interface endpoint?

Reply:
Probably don't need new examples, but this is a required variable so the examples won't work if you don't set this.

Please test the examples since I don't think they'll run in their current state.

Reply:
I already tested it and created a new environment. I am able to run spark jobs from the Tamr VM to the EMR cluster using the EMR service thru the interface endpoint on port 443.

Reply:
I don't think this example works:
https://github.com/fvazquez-caylent/terraform-aws-networking/blob/feature/endpoint-no-sg-module/examples/minimal/vpc.tf

because `interface_endpoint_ingress_sg` is not defined.

Reply:
Dan, you are checking the "minimal" example, I worked with the "complete" one.
https://github.com/fvazquez-caylent/terraform-aws-networking/blob/feature/endpoint-no-sg-module/examples/complete/vpc.tf
Thank you,

Reply:
Right, I can see that you've updated the complete example. 

My point is that I don't think the minimal one works anymore and needs to be updated before we can merge this PR.